.github/workflows/srtool.yml

```diff
@@ -0,0 +1,73 @@
+name: Srtool build
+
+on:
+  push: []
```

Comment:
I suggest changing the action condition to something like:
```
on:
  push:
    branches:
      - 'development/dusty'
```
Or something similar that only triggers for branches where the runtime executable is named `plasm` (Shiden's executable is named `astar-collator`)

Reply:
Thanks @hoonsubin , good Idea!